Create README.md

abdullahibneat opened the pull request:
I used the Reddit "Rootless Pixel Launcher" (https://www.reddit.com/r/Android/comments/6vu2to/rootless_pixel_launcher_20_based_on_aosp_800/) as the body of the README. Only difference is the download link has been moved to the second paragraph.

_(I used the Reddit Enhancement Suite to copy and paste the markdown from the Reddit post to Github)_

abdullahibneat committed:
Create README.md

amirzaidi commented:
Thanks! Could you fix the "Version 2.1 is out" header? If that's fixed I'll merge, can't do it myself right now as I'm not at my PC

abdullahibneat commented:
Will do once I get home (I'm not on PC either XD)

amirzaidi commented:
Thanks a lot for answering on so many issues, I've definitely noticed your help :)

abdullahibneat commented:
No worries, I have some spare time so why not help others out?

RikKoedoot commented:
These kind of People are awesome.
Op zo 27 aug. 2017 om 17:22 schreef Abdullah Ibne Atiq <
notifications@github.com>

> No worries, I have some spare time so why not help others out?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amirzaidi/Launcher3/pull/157#issuecomment-325204942>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcClpP5bG59-PfSlifBSPOUj5MCQkHxsks5scYnRgaJpZM4PDwQJ>
> .
>

abdullahibneat commented:
Thanks! I appreciate that. I'm now back home, I'll see what I can do.

RikKoedoot commented:
are you a developer too?

On Sun, Aug 27, 2017 at 7:57 PM Abdullah Ibne Atiq <notifications@github.com>
wrote:

> Thanks! I appreciate that. I'm now back home, I'll see what I can do.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/amirzaidi/Launcher3/pull/157#issuecomment-325213952>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcClpMH17LR8KBcMRU9Ntg0kmFIz1EFtks5sca4QgaJpZM4PDwQJ>
> .
>

abdullahibneat commented:
I'm afraid I'm not. I'm a 3D modeller student, but I like the modding in Android.

abdullahibneat committed:
Update README.md

Moved latest available version notice under the download section.

abdullahibneat commented:
I updated the content. However, I tried to link a GIF generated from one of your demo videos (this: https://goo.gl/photos/zk6MpwanYhNAMuTn7 to this: https://i.imgur.com/wTCa3uW.gif), but linking from imgur would not work. If you don't mind, can you upload the gif to the code so I can test this? Or if you don't want to, I can simply remove it. Other than that, I moved and fixed the "Version 2.1 is out" right beneath the download section for better visibility, and unified the two FAQs ("FAQ" and "MORE FAQ" from the Reddit post).

abdullahibneat commented:
Basically, this is what I was thinking about: https://github.com/abdullahibneat/test/blob/master/README.md

schliflo commented on README.md:
```diff
@@ -0,0 +1,141 @@
+
+**Pictures**: https://goo.gl/photos/C6kWrgMV3jMfL2Ld9
+
+**Sources**: https://github.com/amirzaidi/Launcher3/commits/o
```
This is unnecessary as we're already on github

abdullahibneat replied:
Maybe just keep the pictures?

schliflo commented on README.md:
```diff
@@ -0,0 +1,141 @@
+# Rootless Pixel Launcher
+
+![Rootless Pixel Launcher in action](https://i.imgur.com/wTCa3uW.gif)
```
Probably better to upload directly to the repo

abdullahibneat replied:
Yes, that's what I suggested amir.

schliflo replied:
you can include it into your pr ;)

abdullahibneat replied:
Found out how to do this.

schliflo commented on README.md:
```diff
@@ -0,0 +1,141 @@
+
+--------
+
+## Version 2.1 is out!
```
Unnecessary -> can easily be seen in releases section

abdullahibneat replied:
I wanted to leave it so that people can check from here what is the latest version. It's fine for me to remove it.

schliflo replied:
The thing is that you have to update the readme with every version then. If @amirzaidi is ok with it thats fine.

abdullahibneat replied:
You're right. Let's wait for his response.

schliflo commented on README.md:
```diff
@@ -0,0 +1,141 @@
+
+- Backport of circle icons
+
+From my previous post:
```
Probably confusing without reddit context

abdullahibneat replied:
Agree. At the time I just copied and pasted from Reddit, so it still needs some work.

abdullahibneat committed:
Update README.md

abdullahibneat committed:
Add files via upload

abdullahibneat committed:
Update README.md

Uploaded GIF

abdullahibneat commented:
All right, I found out how to upload the GIF to pull with the readme.md _(I only use GitHub to host my Jekyll portfolio for free XD)_ and removed the unnecessary "Source" link _(since we're already on GitHub)_.

amirzaidi commented:
Oops, only just now realized you would be shown as the owner of the squashed commit. Oh well, your name is very clearly in the commit now :)

amirzaidi commented:
Great work @abdullahibneat and thanks for the reviewing @schliflo, very much appreciated. Hopefully people will stop asking the same questions in the Issues section now
